charts/datadog/values.yaml

```diff
@@ -319,6 +319,9 @@ datadog:
     ## ref: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows#process-arguments-scrubbing
     stripProcessArguments: false
 
+    # datadog.processAgent.processDiscovery -- Set this to false to disable autodiscovery of integrations
+    processDiscovery: true
```

Comment:
We should probably set the default to `false` until 7.34 is out.

Reply:
Should be fixed now